zaza/openstack/configure/network.py

```diff
@@ -212,12 +212,23 @@ def setup_gateway_ext_port(network_config, keystone_session=None):
     else:
         net_id = None
 
+    # If we're using netplan, we need to add the new interface to the guest
+    current_release = openstack_utils.get_os_release()
+    bionic_queens = openstack_utils.get_os_release('bionic_queens')
+    if current_release >= bionic_queens:
+        logging.warn("Adding second interface for dataport to guest netplan "
+                     "for bionic-queens and later")
+        plumb_guest_nic = True
```

Comment:
Happy to discuss this but I don't think that this is really plumbing in the interface. It's adding it to operating system config. configure_gateway_ext_port, to my mind, does the plumbing when it attaches the port to the guest.